### ISSUE TITLE
v0.5.8 broke the selectize styles

### ISSUE THREAD

**Issue by plicjo** [miguelcobain/ember-cli-selectize#174]
Hello everyone:

I upgraded from `0.5.6` to `0.5.8` this morning and all of the selectize styles were broken.

I reverted back to `0.5.6` and all the styles were fixed again. Did some CSS change between those versions?

Package.json

```
...
 "ember-cli-selectize": "0.5.6",  
...
```

Bower.json

```
...
"ember-cli-shims": "0.1.3",
"selectize": "miguelcobain/selectize.js#master",
...
```

**Comment by miguelcobain** [comment]
Here are the changes between those versions: https://github.com/miguelcobain/ember-cli-selectize/compare/v0.5.6...v0.5.8

Could this be related with #171 ?

It would be helpful if you could show us how to replicate the problem in a new app. Like I say in the reference issue I tried to reproduce and couldn't (there are the steps I used).

I'm mentioning @showy because he was responsible for the "findHost" logic and I'm a bit suspicious that's somehow related?

**Comment by MarioDiar** [comment]
I'm having the same issue. I was able to reproduce the bug in a new application. Tried it in both ember v2.3.0 and v2.8.0.

For now, the workaround is to import the css manually like this until is fixed.

`app.import(app.bowerDirectory + '/selectize/dist/css/selectize.css');`

**Comment by showy** [comment]
@miguelcobain it's in effect the change, I'm looking at it, and there's a basically a typo in the condition which loads the default theme `if (app.theme)` instead of `if (options.theme)` I will create a pull with that modification.

**Comment by miguelcobain** [comment]
This should be fixed in release `0.5.9`.
@showy thanks for pointing out the issue.
